components/FilterDropdownMultiple/index.tsx

```diff
@@ -17,16 +25,83 @@ export default function FilterDropdownMultiple<T>({
   placeholder,
   disabled = false,
 }: FilterDropdownProps<T>) {
+  const handleChange = (selectedOptions: MultiValue<DropdownOption<T>>) => {
+    setStateAction(selectedOptions as DropdownOption<T>[]);
+  };
+
+  // overrides the default MultiValue to display custom text
+  // displays first selected value followed by + n if more than 1 selected
+  // StyledMultiValue appears for each selected option, so if more than 1 is selected,
+  // the rest of the selected options are not shown, instead the + n is shown as part of the first option
+  const StyledMultiValue = ({
+    ...props
+  }: MultiValueProps<
+    DropdownOption<T>,
+    true,
+    GroupBase<DropdownOption<T>>
+  >) => {
+    const { selectProps, data } = props;
+    if (Array.isArray(selectProps.value)) {
+      // find index of the selected option and check if its the first
+      const index = selectProps.value.findIndex(
+        (option: DropdownOption<T>) => option.value === data.value,
+      );
+      const isFirst = index === 0;
+      // find number of remaining selected options
+      const additionalCount = selectProps.value.length - 1;
+
+      return (
+        <P3>
+          {/* display label of first selected option */}
+          {isFirst ? (
+            <>
+              {data.label}
+              {/* display additional count only if more than one option is selected*/}
+              {additionalCount > 0 && ` +${additionalCount}`}
+            </>
+          ) : // don't display anything if not the first selected option
+          null}
+        </P3>
+      );
+    }
+
+    // nothing is selected yet
+    return null;
+  };
+
+  // overrides the default Options to display a checkbox that ticks when option selected
+  const CustomOption = (
+    props: OptionProps<DropdownOption<T>, true, GroupBase<DropdownOption<T>>>,
+  ) => {
+    return (
+      <components.Option {...props}>
+        <StyledOption>
+          <input
+            type="checkbox"
+            checked={props.isSelected}
+            style={{ marginRight: 8 }} // spacing between checkbox and text
```

Comment:
i'm getting a console error: 
"You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`."

perhaps we need to pass an onChange here? 

here's more from the console error
<img width="673" alt="Screen Shot 2024-12-13 at 2 19 48 AM" src="https://github.com/user-attachments/assets/3e8610cd-c89a-4837-bda5-4dc2fef228f3" />